test/parallel/test-net-remote-address-port.js

```diff
@@ -32,13 +32,17 @@ if (common.hasIPv6) remoteAddrCandidates.push('::1', '::ffff:127.0.0.1');
 
 const remoteFamilyCandidates = ['IPv4'];
 if (common.hasIPv6) remoteFamilyCandidates.push('IPv6');
+// Is IPv6 really recognized?
+// REM: assert.ok(common.hasIPv6);
 
 const server = net.createServer(common.mustCall(function(socket) {
   // Test to see real value in CI log
   assert.match(socket.remoteAddress,
                /^127\.0\.0\.1$|^::1$|^::ffff:127\.0\.0\.1$/);
   // REM: assert.ok(remoteAddrCandidates.includes(socket.remoteAddress));
-  assert.ok(remoteFamilyCandidates.includes(socket.remoteFamily));
+  assert.equal(socket.remoteFamily, 'IPv4');
+  assert.equal(socket.remoteFamily, 'IPv6');
```

Comment:
That will fail all the time, as no value can be loosely equals to both `'IPv4'` and `'IPv6'`.

Reply:
```suggestion
if (common.hasIPv6) assert.match(socket.remoteFamily, /^IPv(4|6)$/);
else assert.strictEqual(socket.remoteFamily, 'IPv4');
```

Reply:
I know it would fail all the time, but I would catch it in any case. Anyways, looked better at your example and implemented it. Maybe another CI?